tests/target/issue_3868.rs

```diff
@@ -4,4 +4,6 @@ fn bar() {
     for _ in 0..1 {}
 }
 
-fn baz() {}
+fn baz() {
+    ();
```

Comment:
We were only doing this before because at the time this issue was resolved the rustc parser/AST didn't allow us to differentiate between these empty tuple exprs in statement position and a plain old empty semi `;`